README.md

```diff
@@ -54,9 +55,9 @@ The application will now be available from your browser at http://localhost:8081
    ```
    And
    ```
-   + <feature name="nexus-repository-apt" description="net.staticsnow:nexus-repository-apt" version="1.0.8">
+   + <feature name="nexus-repository-apt" description="net.staticsnow:nexus-repository-apt" version="1.0.9">
    +     <details>net.staticsnow:nexus-repository-apt</details>
-   +     <bundle>mvn:net.staticsnow/nexus-repository-apt/1.0.8</bundle>
+   +     <bundle>mvn:net.staticsnow/nexus-repository-apt/1.0.9</bundle>
    +     <bundle>mvn:org.apache.commons/commons-compress/1.16.1</bundle>
```

Comment:
I think commons-compress version should be `1.18`.

Reply:
Great catch: 4d852e1